docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+# Personal Access Token
+
+The API ML allows generating, validating and invalidating a **Personal Access Token (PAT)** that can be used, for instance, to leverage modern tools such as Version Control Systems (VCS) during
+the development of some application on the z/OS system.
+
+To do that, the API ML offers a set of REST APIs to:
+
+1. [Generate the token](#generate-of-the-token)
```

Comment:
I would structure the APIs to ones used by User and ones used by the administrator.

Reply:
Like directly in this list.